index.html

```diff
@@ -0,0 +1,204 @@
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="highlights">
+            <div class="highlight-1 highlight-item">
+                <div class="top-left-image">
+                    <img src="images/tnw.svg" alt="logo">
+                </div>
+                <div class="top-right-time">
+                    <p>Tuesday - January 26, 2021</p>
+                </div>
+                <div class="lower-moon">
+                    Opal cuts of your apps' internet access to help you stop doomscrolling
+                </div>
+                <div class="lower-moon-2">
+                    by Ivan Mehta <i class="fas fa-clock"></i> <i class="fas fa-share-alt"></i>
+                </div>
+            </div>
+            <div class="highlight-container">
+                <div class="highlight-2 highlight-item">
+                    <div class="lower-moon">
+                        Google unveils privacy friendly AI cookie killer
+                    </div>
+                    <div class="lower-moon-2">
+                        by Thomas Macaulay <i class="fas fa-clock"></i> <i class="fas fa-share-alt"></i>
+                    </div>
+                </div>
+                <div class="highlight-3 highlight-item">
+                    <div class="lower-moon">
+                        This Mac app automatically detects your duplicate files.
+                    </div>
+                    <div class="lower-moon-2">
+                        by Ivan Mehta <i class="fas fa-clock"></i> <i class="fas fa-share-alt"></i>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="latest-funding">
+            <div class="funding">
+                <div class="funding-header">
+                    <a href="#">Latest Funding Rounds On Index.Co <i class="fa fa-angle-right" aria-hidden="true"></i></a>
+                </div>
+                <div class="funding-links">
+                    <a href="#">See More</a>
+                    <a href="#">Get updates By Email</a>
+                    <a href="#" class="third-funding-link">Add info</a>
+                </div>
+
+            </div>
+            <div class="funding-container">
+                <!-- item-1 -->
+                <div class="funding-item">
+                    <div class="one">
+                        <img class="funding-image" src="images/ringover.webp" alt="Ringover">
+                        <h3 class="pad-right">Ringover</h3>
+                        <p class="pad-right"><strong>$12M</strong>
+                            <br>
+                            <span class="sub-head">SERIES A</span>
+                        </p>
+                    </div>
+
+                    <div class="three">
+                        <p><i class="fa fa-tag" aria-hidden="true"></i> Telecommunications</p>
+                        <p><i class="fa fa-briefcase" aria-hidden="true"></i> Expeditions Growth Capital</p>
```

Comment:
These items should be linked (i.e. wrapped inside the `<a>` tag) for all the articles in this section.

Reply:
Done. Thanks for clear feedback.